monitoring/messagesigning/requirements.txt

```diff
@@ -0,0 +1,5 @@
+flask==1.1.2
+loguru
```

Comment:
I see in your other PRs that you used extensively this library, is it a requirement for the feature you are contributing or can we live without it ?
We are making our best to limit the number of dependencies to limit the security risks and size of our docker images. New additions must be motivated.

Reply:
`loguru` is a nice logging library, we started using it because when calling `print`, we wouldn't see that output. `loguru` started logging our output and we saw it was highly configurable.